sql/core/src/test/resources/sql-tests/results/ansi/date.sql.out

```diff
@@ -321,9 +321,10 @@ cannot resolve 'date_add(CAST('2011-11-11' AS DATE), 10.0D)' due to data type mi
 -- !query
 select date_add('2011-11-11', '1')
 -- !query schema
-struct<date_add(2011-11-11, 1):date>
+struct<>
 -- !query output
-2011-11-12
+org.apache.spark.sql.AnalysisException
+cannot resolve 'date_add(CAST('2011-11-11' AS DATE), '1')' due to data type mismatch: argument 2 requires (int or smallint or tinyint) type, however, ''1'' is of string type.; line 1 pos 7
```

Comment:
shouldn't we support it in the ANSI mode?

Reply:
We can't support this since we need to disallow `date + string_literal`